templates/ui-frameworks/vue/field-types/u8/Slider/edit/render.hbs

```diff
@@ -0,0 +1,2 @@
+<label for="{{label}}">{{label}}</label>
+<input name=""{{label}} type="range" min="0" max="255" v-model="{{variable_to_read}}" />
```

Comment:
```suggestion
<input name="{{label}}" type="range" min="0" max="255" v-model="{{variable_to_read}}" />
```

Reply:
Looks my editor did a little bit of bad formatting, thanks for catching this

Reply:
Turns out it wasn't an autoformat issue afterall, it seems like it was an issue with other existing `vue` and `svelte` field types, this must have went unnoticed in previous code reviews